tide_core.module

```diff
@@ -412,13 +406,11 @@ function tide_core_node_grants(AccountInterface $account, $op) {
  */
 function tide_core_node_access(NodeInterface $node, $op, AccountInterface $account) {
```

Comment:
@vincent-gao if this was here because of PBT, should the whole hook go?

Reply:
hey @anthony-malkoun . this was for https://digital-engagement.atlassian.net/browse/SRM-24 , but because of PBT.

Reply:
So this hook is required for sites that don't have PBT? I'm just not sure what we're trying to achieve here.